xcode/Subconscious/Shared/Components/Detail/UserProfileDetailView.swift

```diff
@@ -1078,26 +1017,17 @@ struct UserProfileDetailModel: ModelProtocol {
                 "did:": identity.description
             ]
         )
+        
         return update(
             state: state,
             actions: [
                 .presentUnfollowConfirmation(false),
-                .refresh(forceSync: true)
+                .refresh(forceSync: false)
```

Comment:
Intentional?

Reply:
Yep, this is handled by the AppStore reacting to these.